.github/workflows/github-cxx-qt-tests.yml

```diff
@@ -155,6 +155,7 @@ jobs:
       run: >-
         sudo apt-get update &&
         sudo apt-get install -y
+        gcc-12
```

Comment:
does this mean we require specifically gcc 12 ? why not just gcc / build-essential ?

Reply:
yes, read the commit message

Reply:
OK, i see, maybe we should have a comment above this stating that isn't an issue on ubuntu with GCC 11. But not that we require GCC 12+ etc.

Reply:
Well, it is an issue with GCC 11 on Ubuntu. GCC 11 on Ubuntu only works if lld (or likely mold as well) is used, but not with GNU ld.

Reply:
Right, so just put a comment above, so when we upgrade to Ubuntu 24.04 we know we can probably drop it for gcc.